operations/services.md

```diff
@@ -112,13 +112,14 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `Id` | string | required | Unique identifier of the service. |
 | `EnterpriseId` | string | required | Unique identifier of the [Enterprise](enterprises.md#enterprise). |
 | `IsActive` | boolean | required | Whether the service is still active. |
-| `Name` | string | required | Name of the service. |
+| `Names` | object | required | All translations of the name. |
```

Comment:
The `object` type is wrong. Any idea why?

My guess is because the used type `LocalizedString` is from  `Mews.Library.Dto`  🤔

Reply:
I don't know why it doesn't have its own type in OAS, maybe because it derives from dictionary. We might want to migrate away from the common DTO class for this.